client/src/components/Login.vue

```diff
@@ -6,26 +6,26 @@
       </div>
       <div class="right">
         <h1 style="margin-bottom: 50px;">Redmine kanban board</h1>
-        <p>A belépéshez add meg az API kulcsot. Ezt az alábbi linken éred el:</p>
-        <a href="https://redmine.tigra.hu/my/account">Publikus API kulcs linkje</a> 
+        <p>Fill in API key for login. API key is available at the link below:</p>
+        <a :href="apiKeyUrl">Public API key link</a>
         <div style="margin-top: 80px;">
-          <h4>Bejelentkezés</h4>
+          <h4>Log In</h4>
           <div class="omrs-input-group">
             <label class="omrs-input-underlined">
               <input required v-model="apiKey" id="api-token" name="api-token" type="text">
-              <span class="omrs-input-label">API kulcs</span>
+              <span class="omrs-input-label">API key</span>
             </label>
           </div>
-          <button>BEJELENTKEZÉS</button>
+          <button>LOG IN</button>
         </div>
       </div>
     </form>
   </article>
-  <div v-bind:class="{ active: isActive }" class="toast" id="errorToast">Sikertelen bejelentkezés</div>
+  <div v-bind:class="{ active: isActive }" class="toast" id="errorToast">Login failed</div>
 </template>
 
 <script>
-import { ref } from 'vue'
+  import {onMounted, ref} from 'vue'
```

Comment:
why indent this line? 
also as a solution to always ask for the base url: great!
but i would avoid using mounted for this, i would only make this request once the user clicks the link, otherwise, we are making an extra request for an information not user does not use (or only once or twice)